runtime/value.go

```diff
@@ -76,6 +76,10 @@ func ifaceType(iface interface{}) uintptr {
 	return *(*uintptr)(unsafe.Pointer(&iface))
 }
 
+func ifacePtr(iface interface{}) uintptr {
```

Comment:
This is horrid :|

Reply:
In its defence I benchmarked this - it made a significant speed difference.  It's only used in Value.Equals() and Value.IsNaN()